server/src/main/java/org/elasticsearch/index/engine/LiveVersionMapArchive.java

```diff
@@ -42,21 +42,21 @@ default boolean isUnsafe() {
     /**
      * Returns the total memory usage if the Archive.
      */
-    default long getMemoryBytesUsed() {
+    default long getRamBytesUsed() {
```

Comment:
This change will break stateless build. We can add it as a new default interface method that delegates to the existing method to avoid breakage. It then needs update from the stateless side and finally removing the existing methods from stateful. Not sure if it's worth these dances for just name changes.

Reply:
I am aware of that. I don't think it is worth it. Sure, for a little bit it won't compile, and we should avoid it if we can, but i don't think I'll go that far for a rename!